doc/functions/cycle.rst

```diff
@@ -23,34 +23,33 @@ The ``cycle`` function cycles on a sequence or mapping:
         
     #}
 
-The array can contain any number of values:
+The ``cycle`` function takes two arguments: the ``sequence`` of values to cycle through and the ``position`` in the sequence.
+
+The ``sequence`` must not be empty, the provided values can be of any type:
 
 .. code-block:: twig
 
-    {% set fruits = ['apple', 'orange', 'citrus'] %}
```

Comment:
I like concrete examples instead of random ones. Let's revert to the `fruits` example.

Reply:
I keep this open: i'll rework it later today.

Reply:
No time nor inspiration these last day so ... back to fruits :)